src/discussions/post-comments/PostCommentsView.test.jsx

```diff
@@ -491,8 +550,10 @@ describe('ThreadView', () => {
   });
 
   describe('for discussion thread', () => {
+    const findLoadMoreCommentsButton = () => screen.findByTestId('load-more-comments');
```

Comment:
rename findLoadMoreCommentsButton to "loadMoreCommentsButton"

Reply:
I think with find, the methods represents it functionality